example/Benchmark_kokkos.cpp

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
Authorship is good but this file should have the same triad national security license blurb at the top of this file